packages/client/test/rpc/websocket.spec.ts

```diff
@@ -54,7 +54,7 @@ describe('JSON-RPC call', () => {
         resolve(undefined)
       })
     })
-  }, 3000)
+  }, 30000)
```

Comment:
This specific test was timing out on this branch in the CI (it was passing locally).